SQL/Cleanup_patches/2019-05-15-Reliability_link.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM LorisMenu WHERE Link='reliability/';
```

Comment:
Should this one be done based on the label? If anyone has Reliability in their menu it's likely to be old enough to be an old test_name= style link.

Reply:
@driusan I put both in case someone had renamed the menu item